src/components/pages/CompanySubscriptions/index.tsx

```diff
@@ -133,34 +165,38 @@ export default function CompanySubscriptions() {
 
   const companySubscriptionsCols = CompanySubscriptionsTableColumns(
     t,
-    handleOverlay
+    handleOverlay,
+    currentActive
   )
 
-  return (
-    <main className="page-main-container">
-      {showUnsubscribeOverlay && (
-        <UnSubscribeOverlay
-          openDialog={showUnsubscribeOverlay}
-          handleOverlayClose={() => {
-            setShowUnsubscribeOverlay(false)
-            setLoading(false)
-            setEnableErrorMessage(false)
-          }}
-          handleConfirmClick={() => onUnsubscribeSubmit()}
-          loading={loading}
-          appId={appId}
-          subscriptionId={subscriptionId}
-          enableErrorMessage={enableErrorMessage}
-        />
-      )}
+  const getIcon = (num: number) => {
+    return (
       <Typography
-        className="subTitle"
-        variant="h2"
-        sx={{ pb: 3, marginTop: '35px' }}
+        variant="label3"
+        sx={{
+          background: currentActive + 1 === num ? '#0f71cb' : 'white',
+          color: currentActive + 1 === num ? 'white' : '#0f71cb',
+          outline: '2px solid #0f71cb',
+          flex: '0',
+          marginRight: '20px',
+          borderRadius: '50%',
+          height: '20px',
+          width: '20px',
+          minWidth: '20px',
+          textAlign: 'center',
+          lineHeight: '20px',
+          position: 'relative',
+        }}
       >
-        {t('content.companySubscriptions.headertitle')}
+        {num}
       </Typography>
+    )
+  }
 
+  // Add an ESLint exception until there is a solution
+  // eslint-disable-next-line
+  const renderTable = (query: any) => {
+    return (
```

Comment:
unnecessary  `{ return` 
```
const renderTable = (query: any) => ( <div>...</div> )
```
should be enough

Reply:
got it. will remove it